lib/nexmo_developer/app/services/translator/smartling_downloader.rb

```diff
@@ -0,0 +1,55 @@
+module Translator
+  class SmartlingDownloader
+    include ::Translator::Utils
+
+    attr_reader :file_uris
+
+    def self.call(attrs = {})
+      new(attrs).call
+    end
+
+    def initialize(file_uris: [])
```

Comment:
unfortunately using `[]` as a default value for the param won't work, we need to use `nil` here.
Otherwise, `@file_uris = file_uris || file_paths` will always evaluate to `[]` when the param is not present.